src/olympia/migrations/1129-add-apikeyconfirmation.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `api_apikeyconfirmation` (
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `user_id` integer NOT NULL PRIMARY KEY,
```

Comment:
I'm confused by historical migrations, some mention `user_id` to be unsigned but newer ones don't do that… so I think we should be safe here :thinking:

Reply:
Yeah it shouldn't be `UNSIGNED` - it's not in dev/stage/prod schema at least, and we don't use our `PositiveAutoField` on `UserProfile`.